data/mods/Magiclysm/items/enchanted_bracelet.json

```
@@ -0,0 +1,108 @@
+[
+  {
+    "abstract": "platinum_bracelet_magic",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "platinum bracelet" },
+    "description": "A bracelet, completely covered with a large amount of runes from both sides.",
+    "weight": "50 g",
+    "volume": "25 ml",
+    "material": [ "platinum", "crystallized_mana" ],
+    "symbol": "[",
+    "color": "white",
+    "sided": true,
+    "flags": [ "FANCY", "BELTED", "NO_WEAR_EFFECT" ],
+    "armor": [ { "coverage": 50, "covers": [ "hand_l", "hand_r" ], "specifically_covers": [ "hand_wrist_l", "hand_wrist_r" ] } ]
+  },
+  {
+    "copy-from": "platinum_bracelet_magic",
+    "id": "missile_volley_bracelet",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "missile volley bracelet" },
+    "description": "A bracelet, completely covered with a large amount of runes from both sides.  Wizards love to name it \"ultima ratio\", describing his colossal might.  Require almost a whole week to restore his charge.",
```

```suggestion
    "description": "A bracelet, completely covered with a large amount of runes from both sides.  Wizards love to name it \"Ultima Ratio\", describing its colossal might.  Require almost a whole week to restore its charge.",
```
I don't think rings have a sex 🤔